### PR TITLE
Link was broken

### PR DESCRIPTION
**Pull request by biafra23**
Not sure how to set a releative link. So I mae an absolute link.

### CODE REVIEW

File: docs/tutorial_tx_service_initiate_sign.md
Comment (rmeissner) [inline docs/tutorial_tx_service_initiate_sign.md:26]: It should be similar to how it is done here https://github.com/gnosis/safe-docs/blob/develop/docs/tutorial_tx_service_set_delegate.md .... so basically you were missing the `.md` at the end

**Review by rmeissner — CHANGES_REQUESTED** [review]

File: docs/tutorial_tx_service_initiate_sign.md
Comment (Uxio0) [reply docs/tutorial_tx_service_initiate_sign.md]: I think it should be alright now @rmeissner, I changed both links

**Review by tschubotz — APPROVED** [review]

**Comment by Uxio0** [comment]
Waiting for @rmeissner approval to merge, as he's still requesting changes 😁

**Pull request merged**
